### PR TITLE
PluginBundle and Hook support

### PR DESCRIPTION
**Pull request by xphere**
Initial support for simple plugins.
The `HookSystem` allows listening to "hooks", which can be executed via PHP or through a `hook` Twig function.

### CODE REVIEW

File: src/Elcodi/Component/Plugin/Adapter/EventDispatcher/HookSystemAdapter.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+ * @author Berny Cantos <be@rny.cc>
+ */
+class HookSystemAdapter implements HookSystemInterface
+{
+    /**
+     * @var EventDispatcherInterface
+     *
+     * EventDispatcher which will dispatch the events
+     */
+    protected $dispatcher;
```
Comment (mmoreram) [inline src/Elcodi/Component/Plugin/Adapter/EventDispatcher/HookSystemAdapter.php]: Maybe `$eventDispatcher`?

File: src/Elcodi/Bundle/PluginBundle/composer.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        },
+        {
+            "name": "Elcodi Community",
+            "homepage": "https://github.com/elcodi/PluginBundle/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=5.4",
+
+        "symfony/": "~2.6",
```
Comment (mmoreram) [inline src/Elcodi/Bundle/PluginBundle/composer.json]: w00t? XD

**Comment by mmoreram** [comment]
Once these changes are solved, we can merge that :)

**Comment by xphere** [comment]
Fixed and ready to go.

File: src/Elcodi/Bundle/PluginBundle/composer.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        },
+        {
+            "name": "Elcodi Community",
+            "homepage": "https://github.com/elcodi/PluginBundle/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=5.4",
+
+        "symfony/dependency-injection": "~2.6",
```
Comment (mmoreram) [inline src/Elcodi/Bundle/PluginBundle/composer.json]: "symfony/http-kernel": "~2.6",
 "symfony/config": "~2.6",
 "symfony/dependency-injection": "~2.6",

**Pull request merged**